src/overlays/actors/ovl_Obj_Tokei_Step/z_obj_tokei_step.h

```diff
@@ -5,9 +5,20 @@
 
 struct ObjTokeiStep;
 
+typedef void (*ObjTokeiStepFunc)(struct ObjTokeiStep*, struct GlobalContext*);
+
+typedef struct ObjTokeiStepStep {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ f32 posChangeY;
+    /* 0x10 */ s16 startFallingTimer; // Timer for when a step starts falling after the previous step has started bouncing during opening cutscene
+    /* 0x12 */ s8 numBounces; // Counter for times the step falls below its final Y pos, and thus will bounce back up during opening cutscene
+    /* 0x13 */ s8 hasSoundPlayed;
+} ObjTokeiStepStep; // size = 0x14
```

Comment:
I can't think of a better name off the top of my head, but I think this name has the potential to be confusing

Reply:
I debated about this but just couldn't come up with a better name. Possibly `ObjTokeiStepPanel`. Anyone else got any ideas?

Reply:
`ObjTokeiStepPanel` is better than `ObjTokeiStepStep`